database/dbPost.js

```diff
@@ -9,30 +9,30 @@ const pool = new Pool({
 const getRecs = (pid, callback) => {
   pool.query(`select department, category, subcategory, brand from info where id = ${pid}`)
     .then((categories) => {
-      console.log('response.rows[0]:', categories.rows[0]);
+      // console.log('response.rows[0]:', categories.rows[0]);
```

Comment:
Also here, using promise.all() would make this code cleaner maybe?
https://v8.dev/blog/fast-async

According to the above link, async/await has better performance than a promise. So Later you might want to write your code in async/await. Just for FYI.

Great job btw!